SalaryService.md

```diff
@@ -0,0 +1,104 @@
+Salary Service
+====================
+
+Table of Contents
+_____________
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+Summary
+-----------
+The Salary service returns salary info for a carotene id and, optionally, a postal or a cbsa code in either hourly or yearly amounts.
+
+e.g.: How much does a `Customer service representative` make per `year` in `Chicago`?
+
+The Salary service is available at `/core/salary`.
+
+Request Structure
+-----------
+
+Requests consist of:
+
+| Parameter | Require/Optional | Type | Description |
+|:----------|:-------|:-------|:-------|
+|`carotene_id`| Yes | string | A unique id|
+|`cbsa_code` | Optional | string |  Unique Identifiers for US Metropolitan Statistical areas. |
+|`postal_code` | Optional | string |  Optional. The post code, postal code, or ZIP Code of an address.|
+|`salary_period` | Yes | string | Possible values are "Year", "Hour". | 
+|`country` | Yes | string |A country name or two-letter ISO-3166 country code. Supports only "US". |
```

Comment:
I don't think we have any intention of ever supporting full country names (at least not currently). We should probably just say 'An ISO-3166 country code.'